packages/core/src/constants.ts

```diff
@@ -63,11 +63,22 @@ export const CONTENTFUL_SECTION_ID = 'contentful-section'; // TODO: remove me on
 export const CONTENTFUL_CONTAINER_ID = 'contentful-container';
 export const CONTENTFUL_COLUMNS_ID = 'contentful-columns';
 export const CONTENTFUL_SINGLE_COLUMN_ID = 'contentful-single-column';
+export const CONTENTFUL_BUTTON_ID = 'button';
```

Comment:
[nitpick]
It's by no means necessary but for looking at the persisted data and SDK messages, it would be helpful to use the same prefix as for the container
```suggestion
export const CONTENTFUL_BUTTON_ID = 'contentful-button';
```

(same below)

Reply:
I think it's intentional to keep the optional built-in component IDs more generic than the structure components in case the customer application wants to register their own button to override the contentful built-in button.

If we change our built-in component's IDs to have the `contentful-` prefix, most likely the customer would never override our button (since they would have to register their component as `contentful-button` to do that.  Instead, they would have to disable/opt-out of the contentful button.

Thoughts?

Reply:
I think with the ability to disable the builtins now, that overriding is less important than it was. If we kept the current names, I could see a scenario where a user registers their own button, but gets confused on why our button suddenly disappeared. Prefixing the component names with something would help solve that, and its more understood how components would show up with the disable feature.

Reply:
If we change the built-in component ids it will probably break experiences that have added these components.  I'll test that and see what happens later today.

Reply:
Renaming component ids does impact experiences. I think I'll skip changing the ids for now.

![Screenshot 2024-02-01 at 3 06 37 PM](https://github.com/contentful/experience-builder/assets/8539634/a90c1bad-8c28-4f31-8fc6-8f566ca99162)